docs/pages/link.js

```diff
@@ -183,14 +66,15 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
   </Box>
 </Box>
 `}
-      />
-      <Example
-        id="tab"
-        description={`
-    Use accessibilitySelected and role when using it as a Tab.
-  `}
-        name="Accessible Tab Link"
-        defaultCode={`
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Accessible Tab Link"
+          description="Use accessibilitySelected and role when using it as a Tab."
```

Comment:
Can you add backticks around the prop names? Also "it" isn't clear, and "Tab" doesn't need to be capitalized since we're not referring to our component:
"Use `accessibilitySelected` and `role="tab"` when using Link as a tab."

Reply:
DONE